### ISSUE TITLE
Check why ETS on Beta is not completing Test wcs1-0-0:getcoverage_operations-getcoverage_request-parameter-get-kvp-3 

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ets-wcs10#59]
Recent updates to Beta are showing a slight difference with the Production results.

Test `wcs1-0-0:getcoverage_operations-getcoverage_request-parameter-get-kvp-3 `

The result on Beta is "Continue (did not complete)" but on production it is "PASSED".

The instance under test (IUT) is https://demo.luciad.com/LuciadFusionPlatform/ogc/wcs/compliance_test?service=WCS&request=GetCapabilities&VERSION=1.0.0

The IUT is managed by @robinhoutmeyers

**Comment by dstenger** [comment]
Erroneous expressions were fixed by PR #60.

However, now, a new error occurs when using above referenced iut:
* Output of web browser:
```
There are errors in the configuration of TEAM Engine

org.xml.sax.SAXParseException; lineNumber: 122; columnNumber: 36; Character reference "" is an invalid XML character.
```

* Tomcat log output:
```
518720> [Fatal Error] log.xml:122:36: Character reference "&#8" is an invalid XML character.
518720> [Fatal Error] report_logs.xml:1:6993: Error attempting to parse XML file (href='/root/te_base/users/ogctest/s0003/d1e330_1/d1e1823_1/log.xml').
518720> Error on line 122 column 36
518720>   SXXP0003: Error reported by XML parser: Character reference "&#8" is an invalid XML character.
```

* TEAM Engine log output (until line 122):
```
<log mode="0">
<starttest local-name="getcoverage_operations-getcoverage_request-parameter-post-xml-3" prefix="wcs1-0-0" namespace-uri="http://www.opengis.net/wcs" type="Mandatory" defaultResult="1" path="s0003/d1e330_1/d1e1823_1" file="/root/te_base/work/_root_te_base_scripts_wcs_1.0.0_ctl_/wcs/wcs1-0-0$getcoverage_operations-getcoverage_request-parameter-post-xml-3.test">
<assertion>When a GetCoverage request is made with a PARAMETER name that is listed in the range set, but PARAMETER signed a value that is not defined in AxisDescription, the server returns service exception.</assertion>
<params xmlns:parsers="http://www.occamlab.com/te/parsers"
        xmlns:tec="java:com.occamlab.te.TECore"
        xmlns:tems="java:com.occamlab.te.web.MonitorServlet"
        xmlns:saxon="http://saxon.sf.net/"
        xmlns:xs="http://www.w3.org/2001/XMLSchema"
        xmlns:ogc="http://www.opengis.net/ogc"
        xmlns:ctl="http://www.occamlab.com/ctl"
        xmlns:wcs1-0-0="http://www.opengis.net/wcs"
        xmlns:citef="http://www.opengis.net/cite/functions"
        xmlns:wcs="http://www.opengis.net/wcs"
        xmlns:gen="java:com.occamlab.te.Generator"
        xmlns:gml="http://www.opengis.net/gml"
        xmlns:te="http://www.occamlab.com/te"
        xmlns:fn="http://www.w3.org/2005/xpath-functions"
        xmlns:xlink="http://www.w3.org/1999/xlink">
   <param local-name="VAR_WCS_VERSION" namespace-uri="" prefix="" type="document-node()">
      <value>1.0.0</value>
   </param>
   <param local-name="VAR_WCS_COVERAGE_1" namespace-uri="" prefix=""
          type="document-node()">
      <value>US_Relief</value>
   </param>
   <param local-name="VAR_WCS_RAW_COVERAGE_1" namespace-uri="" prefix=""
          type="document-node()">
      <value>US_Relief</value>
   </param>
   <param local-name="VAR_WCS_COVERAGE_1_WIDTH" namespace-uri="" prefix=""
          type="document-node()">
      <value>200</value>
   </param>
   <param local-name="VAR_WCS_COVERAGE_1_HEIGHT" namespace-uri="" prefix=""
          type="document-node()">
      <value>100</value>
   </param>
   <param local-name="VAR_WCS_DESCRIBE_COVERAGE_HTTP_GET_URL" namespace-uri="" prefix=""
          type="document-node()">
      <value>https://demo.luciad.com:443/LuciadFusionPlatform/ogc/wcs/compliance_test?</value>
   </param>
   <param local-name="VAR_WCS_GET_COVERAGE_HTTP_POST_URL" namespace-uri="" prefix=""
          type="document-node()">
      <value>https://demo.luciad.com:443/LuciadFusionPlatform/ogc/wcs/compliance_test?</value>
   </param>
</params>
</starttest>
<request id="d1e12191_1">
<ctl:request xmlns:ctl="http://www.occamlab.com/ctl" xmlns:parsers="http://www.occamlab.com/te/parsers" xmlns:tec="java:com.occamlab.te.TECore" xmlns:tems="java:com.occamlab.te.web.MonitorServlet" xmlns:saxon="http://saxon.sf.net/" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ogc="http://www.opengis.net/ogc" xmlns:wcs1-0-0="http://www.opengis.net/wcs" xmlns:citef="http://www.opengis.net/cite/functions" xmlns:wcs="http://www.opengis.net/wcs" xmlns:gen="java:com.occamlab.te.Generator" xmlns:gml="http://www.opengis.net/gml" xmlns:te="http://www.occamlab.com/te" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:xlink="http://www.w3.org/1999/xlink"><ctl:url>https://demo.luciad.com:443/LuciadFusionPlatform/ogc/wcs/compliance_test?</ctl:url><ctl:method>get</ctl:method><ctl:param name="VeRsIoN">1.0.0</ctl:param><ctl:param name="SeRvIcE">WCS</ctl:param><ctl:param name="ReQuEsT">DescribeCoverage</ctl:param><ctl:param name="CoVeRaGe">US_Relief</ctl:param></ctl:request>
<response><parser/><content><CoverageDescription xmlns="http://www.opengis.net/wcs" xmlns:gml="http://www.opengis.net/gml" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0.0" xsi:schemaLocation="http://www.opengis.net/wcs https://demo.luciad.com:443/LuciadFusionPlatform/schemas/wcs/1.0.0/describeCoverage.xsd">
  <CoverageOffering>
    <description/>
    <name>US_Relief</name>
    <label>US Relief</label>
    <lonLatEnvelope>
      <gml:pos>-168.1 13.0</gml:pos>
      <gml:pos>-60.099999999999994 75.0</gml:pos>
    </lonLatEnvelope>
    <domainSet>
      <spatialDomain>
        <gml:Envelope srsName="EPSG:4326">
          <gml:pos>-168.1 13.0</gml:pos>
          <gml:pos>-60.099999999999994 75.0</gml:pos>
        </gml:Envelope>
        <gml:Envelope srsName="urn:ogc:def:crs:OGC:1.3:CRS84">
          <gml:pos>-168.1 13.0</gml:pos>
          <gml:pos>-60.099999999999994 75.0</gml:pos>
        </gml:Envelope>
        <gml:RectifiedGrid dimension="2" srsName="urn:ogc:def:crs:OGC:1.3:CRS84">
          <gml:limits>
            <gml:GridEnvelope>
              <gml:low>0 0</gml:low>              <gml:high>1295 743</gml:high>            </gml:GridEnvelope>
          </gml:limits>
          <gml:axisName>x</gml:axisName>
          <gml:axisName>y</gml:axisName>
          <gml:origin>
            <gml:pos>-168.05833333333334 74.95833333333333</gml:pos>
          </gml:origin>
          <gml:offsetVector>0.08333333333333333 0.0</gml:offsetVector>
          <gml:offsetVector>0.0 -0.08333333333333333</gml:offsetVector>
        </gml:RectifiedGrid>
      </spatialDomain>
    </domainSet>
    <rangeSet>
      <RangeSet>
        <name>US_Relief</name>
        <label>US_Relief</label>
       <nullValues>
        <singleValue>35</singleValue>
       </nullValues>
      </RangeSet>
    </rangeSet>
    <supportedCRSs>
      <requestCRSs>urn:ogc:def:crs:OGC:1.3:CRS84</requestCRSs>
      <requestCRSs>EPSG:4326</requestCRSs>
      <responseCRSs>urn:ogc:def:crs:OGC:1.3:CRS84</responseCRSs>
    </supportedCRSs>
    <supportedFormats>
      <formats>GeoTIFF</formats>
    </supportedFormats>
    <supportedInterpolations>
      <interpolationMethod>nearest neighbor</interpolationMethod>
      <interpolationMethod>bilinear</interpolationMethod>
    </supportedInterpolations>
  </CoverageOffering>
<!--Response received in [216] milliseconds--></CoverageDescription></content></response>
</request>
<message id="d1e12228_1"><![CDATA[VAR_WCS_TEMP_CRS: EPSG:4326]]></message>
<message id="d1e12241_1"><![CDATA[VAR_WCS_COVERAGE_1_CRS: EPSG:4326]]></message>
<message id="d1e12255_1"><![CDATA[VAR_WCS_FORMAT_1: GeoTIFF]]></message>
<message id="d1e12267_1"><![CDATA[VAR_WCS_COVERAGE_1_POSITION_FIRST: -168.1 13.0]]></message>
<message id="d1e12278_1"><![CDATA[VAR_WCS_COVERAGE_1_POSITION_SECOND: -60.099999999999994 75.0]]></message>
<message id="d1e12291_1"><![CDATA[VAR_WCS_PARAMETER: ]]></message>
<request id="d1e12306_1">
<ctl:request xmlns:ctl="http://www.occamlab.com/ctl" xmlns:parsers="http://www.occamlab.com/te/parsers" xmlns:tec="java:com.occamlab.te.TECore" xmlns:tems="java:com.occamlab.te.web.MonitorServlet" xmlns:saxon="http://saxon.sf.net/" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ogc="http://www.opengis.net/ogc" xmlns:wcs1-0-0="http://www.opengis.net/wcs" xmlns:citef="http://www.opengis.net/cite/functions" xmlns:wcs="http://www.opengis.net/wcs" xmlns:gen="java:com.occamlab.te.Generator" xmlns:gml="http://www.opengis.net/gml" xmlns:te="http://www.occamlab.com/te" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:xlink="http://www.w3.org/1999/xlink"><ctl:url>https://demo.luciad.com:443/LuciadFusionPlatform/ogc/wcs/compliance_test?</ctl:url><ctl:method>post</ctl:method><ctl:header name="Content-type">text/xml</ctl:header><ctl:body>&lt;GetCoverage xmlns="http://www.opengis.net/wcs" xmlns:gml="http://www.opengis.net/gml"  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/wcs http://schemas.opengis.net/wcs/1.0.0/getCoverage.xsd" version="1.0.0" service="WCS" &gt;&lt;sourceCoverage&gt;US_Relief&lt;/sourceCoverage&gt;&lt;domainSubset&gt;&lt;spatialSubset&gt;&lt;gml:Envelope srsName="EPSG:4326"&gt;                 &lt;gml:pos&gt;-168.1 13.0&lt;/gml:pos&gt;&lt;gml:pos&gt;-60.099999999999994 75.0&lt;/gml:pos&gt;
        &lt;/gml:Envelope&gt; &lt;gml:Grid dimension="2"&gt;&lt;gml:limits&gt;&lt;gml:GridEnvelope&gt;&lt;gml:low&gt;0 0 &lt;/gml:low&gt;&lt;gml:high&gt;200 100 &lt;/gml:high&gt;&lt;/gml:GridEnvelope&gt;&lt;/gml:limits&gt;&lt;gml:axisName&gt;width&lt;/gml:axisName&gt;&lt;gml:axisName&gt;height&lt;/gml:axisName&gt;&lt;/gml:Grid&gt;&lt;/spatialSubset&gt;&lt;/domainSubset&gt;&lt;rangeSubset&gt;
        &lt;axisSubset name=""&gt;
                &lt;singleValue&gt;WCS_PARAMETER_VALUE_INVALID&lt;/singleValue&gt;  &lt;/axisSubset&gt;
&lt;/rangeSubset&gt;

&lt;output&gt;&lt;format&gt;GeoTIFF&lt;/format&gt;&lt;/output&gt;&lt;/GetCoverage&gt;</ctl:body></ctl:request>
<response><parser/><content>MM*&#8;&#1;�&#3;d&#1;&#2;&#1&#3;��&#3;&#3;&#1;&#21;&#1;�&#5;�&#1;&#27;&#1;�(&#3;&#1;&#1;@&#1;&#4;&#1;B&#1;&#1;C&#3;&#1;D&#1;&#7;v&#1;E&#1;&#24;1�&#14;
...
```

Thus, it seems that the returned content contains a character which cannot be processed by TEAM Engine.

**Comment by dstenger** [comment]
@robinhoutmeyers We used your instance (see description of this issue) to test the fix.
As the fix activated new tests, there might be additional failures.
So, you are welcome to review your implementation with the current status of [Beta environment](https://cite.ogc.org/te2/) which contains this fix.
